queries/zathurarc/highlights.scm

```diff
@@ -0,0 +1,37 @@
+; General
+(path) @module
+
+(option) @variable
+
+(command) @keyword
+
+(mode) @variable
+
+(mode_name) @type.builtin
+
+(key) @variable
+
+(key_name) @constant
```

Comment:
Unrelated note: shouldn't vim keycodes should also be highlighted as `@constant`?

Reply:
Should I add the vim keycode highlight to #5942 since it's still open? @clason